client/src/containers/NotRegisteredPage.tsx

```diff
@@ -32,6 +33,10 @@ export const SocialShareForNotRegisteredPage = () => (
   </div>
 );
 
+const JFCLLinkInternal: React.FC<any> = (props) => {
```

Comment:
since we seem to be redeclaring `JFCLLinkInternal` in several files, thoughts on moving it to a `helper.ts` and just importing it to these pages?

Reply:
Yeah totally agree 👍 .

Reply:
Actually, maybe it should just be in the /components folder? `helper.ts` is mostly utility functions and react hooks.

What do you think?

Reply:
to clarify, I was thinking we can make a new `helper.ts` (or something equivalent) in the `/components` folder. are you suggesting we put this function in a standalone file (i.e. `JFCLLinkInternal.tsx`)? that makes sense to me too!

Reply:
Ah, I see. 

Yeah, I was thinking of just putting it in a standalone file. There's already a couple of other small components in there, like `CloseButton.tsx`.

Reply:
that sounds good to me! 🚀

Reply:
Okay, moved this to it's own file!